src/index.js

```diff
@@ -39,6 +39,8 @@ export default function prepass(
 		let c = vnode.__c = new Component(props, context);
 		c.__v = vnode;
 
+		options._skipEffects = options.__s = true;
```

Comment:
Do we want to reset this after the render has run to its original value/undefined? Wondering mainly because of other processes but I don't think it's going to introduce breaking stuff, should not carry over but let's do it for safety